pkg/server/helpers_selinux_test.go

```diff
@@ -157,13 +148,11 @@ func TestCheckSelinuxLevel(t *testing.T) {
 		},
 	} {
 		t.Run(desc, func(t *testing.T) {
-			ok, err := checkSelinuxLevel(test.level)
+			err := checkSelinuxLevel(test.level)
```

Comment:
Since you changed the logic of this function, the following code needs to be updated:

```go
 err := checkSelinuxLevel(test.level)
              if test.expectNoMatch {
                 assert.Error(t, err)                                                                                                                                               
              } else {
                  assert.NoError(t, err)
              }   
```